owrrm/src/main/java/com/facebook/openwifi/rrm/modules/Modeler.java

```diff
@@ -277,7 +277,8 @@ private void fetchInitialData() {
 			JsonObject state = records.data.get(0).data;
 			if (state != null) {
 				try {
-					State stateModel = gson.fromJson(state, State.class);
+					StateInfo stateModel =
+						gson.fromJson(state, StateInfo.class);
```

Comment:
add a timestamp here too?

Reply:
Yep, but I cannot figure out where the timestamp is.. Do you have any thoughts?

Reply:
Should be in `records.get(i).recorded`, but verify for yourself